### PR TITLE
Scopes to JSX

### PR DESCRIPTION
**Pull request by ksaldana1**
Associated Issue: #3506 

### Summary of Changes

* Replace all React.createElement calls to JSX in Scopes.js

### Test Plan

Desk testing. No snapshot test to confirm against. Are we wanting to increase our snapshot coverage going forward? 

### Screenshots/Videos (OPTIONAL)
screen shot 2017-08-09 at 4 20 42 pm

### CODE REVIEW

File: src/components/SecondaryPanes/Scopes.js [hunk 1]
```diff
@@ -11,14 +11,18 @@ import {
...
 
-import _ObjectInspector from "../shared/ObjectInspector";
-const ObjectInspector = createFactory(_ObjectInspector);
+import ObjectInspector from "../shared/ObjectInspector";
 
 import "./Scopes.css";
 
 function info(text) {
-  return dom.div({ className: "pane-info" }, text);
+  return (
```
Comment (wldcordeiro) [inline src/components/SecondaryPanes/Scopes.js]: Hmm, to be more React idiomatic let's do.

```js
function Info({ text }) // or PaneInfo
```

File: src/components/SecondaryPanes/Scopes.js [hunk 2]
```diff
@@ -57,16 +61,18 @@ class Scopes extends PureComponent {
...
+        />
+      );
     }
-
-    return dom.div(
-      { className: "pane scopes-list" },
-      pauseInfo ? scopeInspector : info(L10N.getStr("scopes.notPaused"))
+    return (
+      <div className="pane scopes-list">
+        {pauseInfo ? scopeInspector : info(L10N.getStr("scopes.notPaused"))}
```
Comment (wldcordeiro) [inline src/components/SecondaryPanes/Scopes.js]: If the above is done this line should be 
```js
{pauseInfo ? scopeInspector : <PaneInfo>{L10N.getStr("scopes.notPaused")}</PaneInfo>}
```

Comment (ksaldana1) [inline src/components/SecondaryPanes/Scopes.js]: Good call. Can actually do the same on ln 62 above.

Comment (wldcordeiro) [inline src/components/SecondaryPanes/Scopes.js]: True you could make this block 

```jsx
<PaneInfo>
  {pauseInfo ? L10N.getStr("scopes.notAvailable") : L10N.getStr("scopes.notPaused")}
</PaneInfo>
```

**Comment by wldcordeiro** [comment]
Thanks a lot @ksaldana1 you've been a big help on the JSX transforms. I've made a tweak to your last commit but this looks ready! I'll wait for CI and merge.

**Comment by ksaldana1** [comment]
I agree, @jasonLaster. Are you wanting snapshot tests or more functional assertion unit tests? I guess the answer to that is probably both, ideally. I would like to become more familiar with Enzyme/Jest snapshots, so would be a good opportunity for me to swing back around on some of these more straightforward components.

**Comment by jasonLaster** [comment]
Yeah. Tests are really for working on components reliably. They're fun to
write too.


For scopes it's just snapshots. The interaction is in the Onject Indpector

**Pull request merged**